spec/tests/framework.fmf

```diff
@@ -27,7 +27,7 @@ description: |
         Exit code and BeakerLib's ``TestResults`` file determine
         the test result.
 
-example:
+example: |
```

Comment:
This are two examples, they should be formatted as two separate sections.

Reply:
@psss how, can you suggest?